internal/bgp/native/backoff.go

```diff
@@ -1,4 +1,4 @@
-package bgp
+package native
```

Comment:
This will be used only as the exposed `session` interface from the controller, however I'd pick a more descriptive package name.
From the calling point of view it's gonna be `native.New(..)`, native.Session, etc which (I think) it's less descriptive than `nativebgp.New()` or something like that.

Reply:
I would go with bgpnative and bgpfrr